src/engraving/layout/layoutsystem.cpp

```diff
@@ -110,6 +110,9 @@ System* LayoutSystem::collectSystem(const LayoutOptions& options, LayoutContext&
         if (system->hasCrossStaffOrModifiedBeams()) {
             updateCrossBeams(system, ctx);
         }
+        if (ctx.curMeasure->isMeasure()) {
```

Comment:
There are now three more-or-less consecutive `if (ctx.curMeasure->isMeasure()) {` blocks. Can't they be unified?

Reply:
Yeah, did a bit of tidying up now 👍